contrail-ui/src/components/main/main-app-bar/MainAppBar.tsx

```diff
@@ -9,7 +9,7 @@ class MainAppBar extends Component<any, any> {
     public render() {
         const { classes } = this.props;
         return (
-            <AppBar position="absolute" className={classes.appBar}>
+            <AppBar position="fixed" className={classes.appBar}>
```

Comment:
can we use class instead of this inline thing? or is this feature of the react

Reply:
It's a built in prop of material-ui, there's also additional behaviour besides the positioning when you set it

Reply:
ahh k